go/vt/srvtopo/resilient_server.go

```diff
@@ -77,19 +77,19 @@ type ResilientServer struct {
 	*SrvKeyspaceNamesQuery
 }
 
+var counts *stats.CountersWithSingleLabel
+
+func init() {
+	counts = stats.NewCountersWithSingleLabel("TabletSrvTopo", "Resilient srvtopo server operations", "type")
+}
+
 // NewResilientServer creates a new ResilientServer
 // based on the provided topo.Server.
-func NewResilientServer(ctx context.Context, base *topo.Server, counterPrefix string) *ResilientServer {
+func NewResilientServer(ctx context.Context, base *topo.Server) *ResilientServer {
 	if srvTopoCacheRefresh > srvTopoCacheTTL {
 		log.Fatalf("srv_topo_cache_refresh must be less than or equal to srv_topo_cache_ttl")
 	}
 
-	metric := ""
-	if counterPrefix != "" {
```

Comment:
`counterPrefix` was never empty, so we never created counters with no name anyway.